src/test/groovy/unit/GatlingPluginTest.groovy

```diff
@@ -78,10 +79,9 @@ class GatlingPluginTest extends GatlingUnitSpec {
 
     def "should create processGatlingResources task"() {
         expect:
-        with(project.tasks.getByName("processGatlingResources")) {
-            it instanceof ProcessResources
-            it != null
-            it.actions.find { it.action instanceof LogbackConfigTaskAction } != null
+        with(project.tasks.getByName("processGatlingResources")) { Task task ->
+            task instanceof ProcessResources
+            task.actions.any { it.hasProperty("action") && it.action instanceof LogbackConfigTaskAction }
```

Comment:
Isn't the `it.hasProperty("action")` part enough to fix the error?

Note: I do prefer to have explicitly named `task`

Reply:
@Isammoc No, we want to make sure the task is a `org.gradle.api.internal.AbstractTask$TaskActionWrapper` and that the wrapped action is a LogbackConfigTaskAction.

Reply:
Gradle may in future add more default task action ( currently - there's 2 actions in the list, one from Gradle, another is ours).
Additionally, plugin users may also add their actions to our taksks via doFirst \ doLast.

So, we have to find all action with `.action` property and them we must check class name as well.